provider_shopper/lib/models/cart.dart

```diff
@@ -40,4 +40,10 @@ class CartModel extends ChangeNotifier {
     // depend on it.
     notifyListeners();
   }
+
+  void remove(Item item) {
+    _itemIds.remove(item.id);
+    // Don't forget to tell dependant widgets to rebuild _every time_ you change the model.
```

Comment:
This is over 80 characters, please add a newline.

Reply:
+1.

nit: "dependent" is the correct spelling.